README.md

```diff
@@ -849,6 +850,19 @@ common: &common
     subscriber:
       topic: "test-topic"
       group_id: ""
+  sqs:
+    subscriber:
```

Comment:
Could we have `Enable`s here? Better to get more info right?

Reply:
@laynax I have added the validation step to check if the QueueURL is present for both publisher and subscriber ✅